starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items=[];
+    this.length=0;
+
+  }
+  add(item) {
+this.items.push(item);
+this.length=this.items.length;
+this.items.sort((a,b) => a-b);
+
+  }
+  get(pos) {
+    //this.length=this.items.length;
+    return this.items[pos-1];
+  }
+  max() {
+    if (this.length>0){
+      return Math.max(...this.items);
+    }
+    else throw new Error("EmptySortedList");
+  }
+  min() {
+    if (this.length>0){
+
+    return Math.min(...this.items);
+  }
+  else throw new Error("EmptySortedList");
+}
+  average() {
+    if(this.length>0){
+    let total = this.items.reduce((a,b) =>a+b);
```

Comment:
you've already calculated the sum, so to help keep your code dry you could call it here 

```
      return this.sum() / this.length;

```